app-android-xamarin/Moodis/Moodis/Resources/layout/activity_group.xml

```diff
@@ -0,0 +1,78 @@
+<android.support.constraint.ConstraintLayout
```

Comment:
This file contains a lot of magic numbers. Put them to dimens.xml. Besides, are those hardcoded numeric values really needed? What would happen if you run the app on smaller screen, will every element be laid out as expected?

Reply:
Oh, and for margins use `@dimen/grid_unit_{some_number}`